src/rust-crypto/libolm_migration.ts

```diff
@@ -158,7 +160,17 @@ async function migrateBaseData(
     // If we have a backup recovery key, we need to try to figure out which backup version it is for.
     // All we can really do is ask the server for the most recent version.
     if (recoveryKey) {
-        const backupInfo = await requestKeyBackupVersion(http);
+        let backupCallDone = false;
+        let backupInfo: KeyBackupInfo | null = null;
+        while (!backupCallDone) {
+            try {
+                backupInfo = await requestKeyBackupVersion(http);
+                backupCallDone = true;
+            } catch (e) {
+                // Retry until successful, use simple constant delay
+                await sleep(2000);
```

Comment:
Figured out it was better to wait a bit, not sure we need exponential backoff though